### DISCUSSION TITLE
Migrazione da H.A. Supervided a H.A. Core.

### DISCUSSION THREAD

**Discussion by lexlutor72** [indomus/forum#3188] [GENERALE - Discussioni generiche]
Ciao a tutti,
volevo chiedere se fosse possibile effettuare un backup del mio H.A. Supervised ed effettuare il restore su H.A. Core o H.A. OS in modalità indolore, funzionando tutto direttamente all'avvio senza problemi. Grazie.

**Comment by snakuzzo** [comment]
Se tiri su HA OS. Puoi semplicemente fare il backup e ripristinarlo al primo boot.

Se decidi di passare a core, non puoi farlo così facilmente. Per la parte core dovresti estrarre dallo zio la directory home Assistant e apportare o dovuti aggiustamenti nei file di configurazione.
Inoltre non avresti tutti gli addon, che dovrai tirare su come servizi a parte e configurarli a mano.

Dalla domanda che hai fatto, immagino tu non sia così pratico. Quindi la soluzione Core per ora la scarterei.
Tira su HA OS e, dopo aver ripristinato il backup sulla nuova installazione, ricordati di fare un reboot del server.

**Reply by lexlutor72** [reply]
Io al momento utilizzo Home Assistant Supervised su Debian con Container. Purtroppo, sovente, supervised non funziona correttamente per colpa del Sistemd journaled che sistematicamente non funziona. Più volte sono riuscito a riparare la configurazione, però comincia a diventare una scocciatura, ecco perché mi chiedevo se ci fosse una installazione che non mi creasse problemi.

**Reply by KC-inDomus** [reply]
allora ti ha già risposto più che abbondantemente snake: Home Assistant OS.

**Reply by lexlutor72** [reply]
Vorrei utilizzare una procedura immediata senza perdita di tempo.......ne ho poco......se HA OS è la soluzione più indolore utilizzerò quella.

**Comment by KC-inDomus** [comment]
> Ciao a tutti, volevo chiedere se fosse possibile effettuare un backup del mio H.A. Supervised ed effettuare il restore su H.A. Core o H.A. OS in modalità indolore, funzionando tutto direttamente all'avvio senza problemi. Grazie.

integro la risposta di snakuzzo dicendo che una cosa la puoi fare: effettuare il backup della "sola" core (impostazioni > sistema > backup e segliere solo home assistant e file di configurazione) e ripristinarla sulla "nuova" core. naturalmente questo ti ripristina il db di home assistant e altro, ma non gli addon - che dovrai implementare a sé.
